src/components/pages/program-view/ProgramViewContainer.jsx

```diff
@@ -55,19 +82,18 @@ const ProgramViewContainer = props => {
       </TitleContainer>
 
       {/*Course Description Section*/}
-      <Container>{programData.programDesc}</Container>
+      <Container>{programData.programDescription}</Container>
 
       {/*Render Course Components Section*/}
-      {/*Will replace with the actual Course component when available*/}
       <Container>
         <h4>Courses</h4>
         {programData.courses.map(data => (
-          <div key={Math.random()}>
-            <div>Course Component</div>
-            <div>{data.courseName}</div>
-            <div>{data.courseDesc}</div>
-            <br />
-          </div>
+          <CourseView
+            key={Math.random()}
```

Comment:
React Docs would ask you not to use `Math.random()` for keys. Instead, we could use `CourseId` or something related to the data these are handling instead of `Math.random()` for keys. That way, our keys are dependent on the data its holding (which is kinda the whole point of keys under the hood). 

And, fun bonus, we get additional error-checking built into our component b/c we don't want to display the same course twice!

Reply:
Excellent indeed yes! Will make the changes to the keys in this component as well. More soon!